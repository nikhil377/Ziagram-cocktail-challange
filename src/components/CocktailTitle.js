import React from 'react'

const CocktailTitle =()=>{

    return(
      <div className="main">
        <div className="cocktail-title">
            <h1 className="title">Welcome to CocktailDB App</h1>
        </div>
      </div>
    )
}

export default CocktailTitle
