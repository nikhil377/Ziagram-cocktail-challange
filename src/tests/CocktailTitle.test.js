// MyComponent.test.js
import React from 'react';
import CocktailTitle from '../components/CocktailTitle';
import { configure ,shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
let wrapper;
beforeEach(()=>{
  wrapper = shallow(<CocktailTitle />);
})
describe("CocktailTitle", () => {
  test('should render the title', () => {
      //console.log(wrapper.debug());
     expect(wrapper.getElements()).toMatchSnapshot();
});
});
  
