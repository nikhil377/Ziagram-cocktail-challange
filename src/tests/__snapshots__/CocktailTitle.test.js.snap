// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CocktailTitle should render the title 1`] = `
Array [
  <div
    className="main"
  >
    <div
      className="cocktail-title"
    >
      <h1
        className="title"
      >
        Welcome to CocktailDB App
      </h1>
    </div>
  </div>,
]
`;
